cmd/start_server/main.go

```diff
@@ -15,9 +15,28 @@
 package main
 
 import (
+	"context"
+	"flag"
+	"log"
+
+	"github.com/googleinterns/cloudai-gcp-test-resource-reaper/pkg/logger"
 	"github.com/googleinterns/cloudai-gcp-test-resource-reaper/pkg/manager"
 )
 
 func main() {
-	manager.StartServer("localhost", "8000")
+	port := flag.String("port", "8000", "port to run gRPC server on")
+	logsProject := flag.String("logs-project", "", "GCP Project ID for where to store logs")
+	logsName := flag.String("logs-name", "", "name of logs")
+
+	flag.Parse()
+
+	if err := logger.CreateLogger(); err != nil {
+		log.Fatal(err)
+	}
+	defer logger.Close()
+	if len(*logsProject) > 0 && len(*logsName) > 0 {
+		logger.AddCloudLogger(context.Background(), *logsProject, *logsName)
```

Comment:
Is it worth adding some logging here about which project is being logged to?

Probably warning level - I think we may also want to log at warning if no project is provided.

Thoughts?

Reply:
Added log message for which logs and project are being written to